### PR TITLE
Update listing and pricing rules

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the Listing Rules and Pricing Rules sections for ASC 4.0.

## Affected documentation pages

All topics in the Listing Rules and Pricing Rules toc sections.

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

staging build #396 (395 failed due to bad link path)

### CODE REVIEW

File: src/sales-channels/asc/floor-price.md [hunk 1]
```diff
@@ -11,40 +11,40 @@ Sections of an intelligent repricing rule include:
...
-1. For **Floor Price Source**, choose an attribute.
+1. For **Floor Price Source**, choose an price source attribute.
 
-   Select your Magento attribute that will indicate your relative floor limit. For example, if you don’t want your product listing price to go below the Cost of your item, you would choose the Cost attribute.
+   Choose the Magento [product attribute]({% link catalog/product-attributes.md %}) that indicates your relative floor limit. For example, if you do not want your Amazon listing price to go below the cost of your item, you would choose the _Cost_ attribute.
 
 1. For **Floor Price Action**, choose an option.
 
-    - **Decrease By** - Choose when you want the defined Floor Price Source value to be adjusted down, creating a lower floor price for the rule, before listing to Amazon.
+    - **Decrease By** - Choose when you want the defined _-Floor Price Source_ value to be adjusted down, creating a lower floor price for the rule, before listing to Amazon.
```
Comment (leslietilling) [inline src/sales-channels/asc/floor-price.md]:
```suggestion
    - **Decrease By** - Choose when you want the defined _Floor Price Source_ value to be adjusted down, creating a lower floor price for the rule, before listing to Amazon.
```
assuming that `-` was a typo

File: src/sales-channels/asc/listing-rule-preview.md [hunk 2]
```diff
@@ -4,44 +4,41 @@ redirect_from:
...
 
 1. Modify your [Listing Rule Conditions]({% link sales-channels/asc/ob-define-condition-example.md %}).
 
-1. Click **Preview Changes** in the header bar.
+1. Click **Preview Changes**.
 
 1. Review and confirm your listings in the _Ineligible Listings_, _Eligible Listings_, and _New Listings_ tabs.
 
-1. If your listings match your expectations, click **Save Listing Rules**. If your listings do no not display as expected, click **Back** and continue to modify your conditions until your listings match your expectations.
+1. If your listings match your expectations, click **Save and close**. If your listings do no not display as expected, click **Back** and modify your rules and conditions until your listings match your expectations.
```
Comment (leslietilling) [inline src/sales-channels/asc/listing-rule-preview.md]:
```suggestion
1. If your listings match your expectations, click **Save and close**.

    If your listings do no not appear as expected, click **Back** and modify your rules and conditions until your listings match your expectations.
```

File: src/sales-channels/asc/listing-rules.md [hunk 3]
```diff
@@ -5,37 +5,33 @@ redirect_from:
...
 
-Changes to quantity and eligibility status impact all listings that share the Amazon Seller SKU in marketplaces which exist for stores selling in the same region (as defined in the Amazon Marketplace field during [Store Integration]({% link sales-channels/asc/store-integration.md %})) will also be impacted. However, a change to a shared Amazon Seller SKU in one region will not affect the product's Amazon listings in a different region (as defined during Store Integration).
+Changes to quantity and eligibility status impact all listings that share the Amazon Seller SKU in marketplaces which exist for stores selling in the same region (as defined in **Amazon Marketplace Country** during [store integration]({% link sales-channels/asc/store-integration.md %})) will also be impacted. However, a change to a shared Amazon Seller SKU in one region will not affect the product's Amazon listings in a different country.
```
Comment (leslietilling) [inline src/sales-channels/asc/listing-rules.md]:
```suggestion
Changes to quantity and eligibility status impact all listings that share the Amazon Seller SKU in marketplaces that exist for stores selling in the same region (as defined in **Amazon Marketplace Country** during [store integration]({% link sales-channels/asc/store-integration.md %})). However, a change to a shared Amazon Seller SKU in one region will not affect the product's Amazon listings in a different country.
```

File: src/sales-channels/asc/order-settings.md [hunk 4]
```diff
@@ -74,27 +74,10 @@ _Order Settings_
...
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|PendingAvailability|No|Inventory not reserved |Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Canceled|No|Inventory not reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Error|No|Inventory not reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Unshipped|Yes|Inventory reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|PartiallyShipped|Yes|Inventory reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Shipped|Yes|Inventory reserved|Inventory not reserved|
-
 ### Order Creation with Multi Source Inventory (MSI)
 
+*Supported in Magento 2.3.x integrations only.
```
Comment (leslietilling) [inline src/sales-channels/asc/order-settings.md]: The `*` with no space? Not sure what was intended here.

File: src/sales-channels/asc/price-scope.md [hunk 5]
```diff
@@ -3,8 +3,13 @@ title: Price Scope
 ---
 
 
-Magento provides configuration for your pricing scope to be set to global or website. If pricing is set to global scope, there will be a single price source for all websites. If pricing is set to website scope, websites can vary their pricing across and also have a fallback default pricing value. To set your price scope, see [Magento Price configuration]({% link configuration/catalog/catalog.md %}).
+Magento provides configuration for your [pricing scope](https://docs.magento.com/m2/ee/user_guide/configuration/catalog/catalog.html#price) to be set to `Global` or `Website`. If pricing is set to `Global`, there will be a single price source for all websites. If pricing is set to `Website`, your websites can vary their pricing across and also have a fallback default pricing value. See [Catalog Price](https://docs.magento.com/m2/ee/user_guide/configuration/catalog/catalog.html#price).
```
Comment (leslietilling) [inline src/sales-channels/asc/price-scope.md]: Should use new link format here

File: src/sales-channels/asc/pricing-products.md [hunk 6]
```diff
@@ -21,27 +21,25 @@ You can add pricing rules for your Amazon listings. Price rules can be used to a
...
 
 You can control important aspects of rules, such as active/inactive status, website eligibility, optional date ranges, and optional priority levels (used for rule stacking).
 
 For example, you can define and set the conditions for a price rule that, when the conditions are met, automatically adjust your listing price before it is sent to Amazon.
 
-Another pricing option is a price override, which is set at the individual listing level. A price override can be set, and an override ignores/takes priority over all other defaults, settings, and rules. An override can be set for price, handling time, condition, and seller notes (with a few exceptions). See [Overrides]({% link sales-channels/asc/overrides.md %}).
+Another pricing option is a [price override]({% link sales-channels/asc/overrides.md %}), which is set at the individual listing level. A [price override]({% link sales-channels/asc/overrides.md %}) can be set, and an override ignores/takes priority over all other defaults, settings, and rules. An [override]({% link sales-channels/asc/overrides.md %}) can be set for price, handling time, condition, and seller notes (with a few exceptions).
```
Comment (leslietilling) [inline src/sales-channels/asc/pricing-products.md]: two links to the same place in a single paragraph?

File: src/sales-channels/asc/order-settings.md (hunk 4 above)
Comment (hickskenh) [reply src/sales-channels/asc/order-settings.md]: it was an attempt at a note, on the fly, doing too many things at once.   changing.

File: src/sales-channels/asc/price-scope.md (hunk 5 above)
Comment (hickskenh) [reply src/sales-channels/asc/price-scope.md]: done in a followup commit

File: src/sales-channels/asc/pricing-products.md (hunk 6 above)
Comment (hickskenh) [reply src/sales-channels/asc/pricing-products.md]: yes.  since they were so close, i thought it looked odd to link one and not the other.

**Comment by hickskenh** [comment]
running tests

**Pull request merged**